lib/src/posthog_flutter_web_handler.dart

```diff
@@ -90,6 +90,10 @@ Future<dynamic> handleWebMethodCall(MethodCall call, JsObject context) async {
         call.arguments['key'],
       ]);
       break;
+    case 'flush':
+      // not supported on Web
+      // analytics.callMethod('flush');
```

Comment:
@pauldambra is there an alternative to flush on the web?

Reply:
Looks like there is no equivalent: https://github.com/PostHog/posthog-js/issues/205#issuecomment-1022028337

Reply:
No, I'd guess those comments are up to date

Reply:
yeah didn't find in the code either, will merge as it is.